<launch>
    <node respawn="true" pkg="joy"
        type="joy_node" name="urc">
        <param name="dev" type="string" value="/dev/input/js0" />
        <param name="deadzone" value="0.12"/>
    </node>

    <!--    Axes -->
        <param name="axis_linear" value="1" type="int"/>
        <param name="axis_angular" value="0" type="int"/>
        <param name="axis_strafe" value="2" type="int"/>

        <param name="scale_linear" value="1" type="double"/>
        <param name="scale_angular" value="1" type="double"/>
        <param name="scale_strafe" value="1" type="double"/>
        <node pkg="joystick" type="urc" name="joystick"/>

</launch>